### PR TITLE
feat: 🎸 finish codemirror formatter ir

### PR DESCRIPTION
**Pull request by IWANABETHATGUY**
## Summary
1. codemirror grammar formatter ir

## Test Plan
1. Testing locally or see my preview video.

### CODE REVIEW

File: website/playground/src/DesktopPlayground.tsx [hunk 1]
```diff
@@ -120,31 +122,12 @@ export default function DesktopPlayground(
...
-											height: 16,
-											marginRight: 5,
-										}}
-									/>
-								)}
-								<CopyIcon style={{ width: 16, height: 16 }} />
-							</button>
-							<pre className="h-screen overflow-scroll">{formatter_ir}</pre>
+							<CodeMirror
+								value={formatter_ir}
```
Comment (MichaReiser) [inline website/playground/src/DesktopPlayground.tsx:126]: Does the same grammar work for prettiers ir too?

Comment (IWANABETHATGUY) [reply website/playground/src/DesktopPlayground.tsx:126]: If our IR is the same as prettier's, then it should work.

**Comment by IWANABETHATGUY** [comment]
https://user-images.githubusercontent.com/17974631/183020355-be3381a2-d1f0-429f-9122-2b4524f7db03.mov

**Comment by IWANABETHATGUY** [comment]
Source code is here https://github.com/IWANABETHATGUY/codemirror-rome-formatter-ir, feel free to review it.

**Comment by MichaReiser** [comment]
Neat, I love it!

File: website/playground/src/DesktopPlayground.tsx (hunk 1 above)
Comment (MichaReiser) [reply website/playground/src/DesktopPlayground.tsx:126]: Prettier's IR uses a very similar structure: `(` and `)` for an element, `[]` for a sequence of children.

**Review by MichaReiser — APPROVED** [review]

File: website/playground/src/DesktopPlayground.tsx (hunk 1 above)
Comment (IWANABETHATGUY) [reply website/playground/src/DesktopPlayground.tsx:126]: I will replace in next pr

**Pull request merged**